### PR TITLE
ADD sendPasswordResetEmail

### PR DESCRIPTION
**Pull request by sarovin**

### CODE REVIEW

File: src/auth.js [hunk 1]
```diff
@@ -52,6 +52,11 @@ function * createUserWithEmailAndPassword (email, password) {
   return yield call([auth, auth.createUserWithEmailAndPassword], email, password)
 }
 
+function * sendPasswordResetEmail (email) {
```
Comment (n6g7) [inline src/auth.js]: Can we add an optional parameter for `actionCodeSettings`?

File: docs/_reference/auth.md [hunk 2]
```diff
@@ -250,6 +250,33 @@ methods:
       }
       ```
 
+  - signature: auth.sendPasswordResetEmail(email, actionCodeSettings)
+    id: sendPasswordResetEmail
+    generator: true
+    description: You can send a password reset email to a user.
```
Comment (n6g7) [inline docs/_reference/auth.md]: Can you replace with "Sends a password reset email to the given email address.", please? :)

File: docs/_reference/auth.md [hunk 3]
```diff
@@ -250,6 +250,33 @@ methods:
...
+    arguments:
+      - name: email
+        required: true
+        type: String
+        description: The email address with the password to be reset.
+      - name: actionCodeSettings
+        required: false
+        type: [firebase.auth.ActionCodeSettings](https://firebase.google.com/docs/reference/js/firebase.auth.html#.ActionCodeSettings)
+        description: The action code settings.
+    output: A [firebase.Promise](https://firebase.google.com/docs/reference/js/firebase.Promise.html) instance.
```
Comment (n6g7) [inline docs/_reference/auth.md]: Again, no output here. :)

**Comment by sarovin** [comment]
@n6g7 Can we use prettier or the .editorconfig? I have some problems with the lint...

**Comment by sarovin** [comment]
closed for #56

**Pull request closed without merging**